api/development.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16-alpine3.15
```

Comment:
You'll probably need to make this change in a separate PR as it will likely break a tonne of stuff. Also be aware that this must match the production `Dockerfile` at all times, or the app will no longer build.

Reply:
Submitted PR to update node version in #211